src/application.conf

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Let's stick with json extension for json files. "application.conf.json" would be admissible but personally I don't like it.